data/binding/json.go

```diff
@@ -0,0 +1,331 @@
+package binding
+
+import (
+	"errors"
+	"sync"
+
+	"fyne.io/fyne/v2/data/binding"
+
+	jsonvalue "github.com/Andrew-M-C/go.jsonvalue"
+)
+
+// JSONValue supports binding a jsonvalue.V{}
```

Comment:
As noted above, jsonvalue.V isn't part of the API any more, so probably best not in the docs.